Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -213,6 +213,9 @@ MediaPlayerPrivateGStreamer::~MediaPlayerPrivateGStreamer()
         ASSERT(bus);
         gst_bus_set_sync_handler(bus.get(), nullptr, nullptr, nullptr);
         g_signal_handlers_disconnect_matched(m_pipeline.get(), G_SIGNAL_MATCH_DATA, 0, 0, nullptr, nullptr, this);
+        // BRCM specific, flush to unlock aud/vid filters
```

Comment:
I can remove this change from pull request so we can proceed with others. This patch is needed if when brcm decoders limit_buffering is enabled, so you should be good without it.